src/requests/typings/function.ts

```diff
@@ -63,9 +86,16 @@ type FunctionEventHandlers = {
     event: GraphQLQueryRequest
     response: GraphQLQueryResponse
   }
+  [APP_EVENT_FILTER]: {
+    event: AppEventFilterRequest
+    response: AppEventFilterResponse
+  }
 }
 
-export type FunctionEvent = GraphQLFieldTypeMappingRequest | GraphQLQueryRequest
+export type FunctionEvent =
+  | GraphQLFieldTypeMappingRequest
+  | GraphQLQueryRequest
+  | AppEventFilterRequest
```

Comment:
@anho have you folks started to think about the best place to centralize these types and core functions capabilities logic yet? the more i look at this stuff propagating around in so many repos the more i think we need a @contentful/functions-types package or something of that nature very soon

Reply:
Totally on the same page about it, let's brainstorm next week about it.